sensor_stack/sparton/src/sparton.py

```diff
@@ -271,16 +271,24 @@ def getData():
     else:
         print 'Error in opening port'
 
-    r = rospy.Rate(10)
+	if rospy.has_param('/ros_rate'):
+		temp_rate = rospy.get_param('/ros_rate')
+	else:
+		raise RuntimeError("ROSParam '/ros_rate' does not exist.")
+		rospy.signal_shutdown("ROSParam '/ros_rate' does not exist.")
+		sys.exit(0)
+
+	r = rospy.Rate(temp_rate)
+
```

Comment:
@aniket11bh @kshitijgoel007 @VishnuDuttSharma We should change these rates to the global constant right?